### ISSUE TITLE
Issue with Integration starting

### ISSUE THREAD

**Issue by phil-the-dev** [wez/govee-lan-hass#7]
When I first discovered the integration, it worked perfectly. Then, I bought some more Govee accessories, paired via Govee like usual, and then all my other Govee devices stopped working. 

I tried new API keys, tried removed/rebooted/reinstalling the integration, everything I could think of, but nothing got the Integration to work again and doesn't return any devices.

I looked at the logs and I'm seeing the following error:
```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 289, in _async_setup_platform
    await asyncio.shield(task)
  File "/config/custom_components/govee_lan/light.py", line 150, in async_setup_entry
    await controller.query_http_devices()
  File "/usr/local/lib/python3.10/site-packages/govee_led_wez/govee.py", line 228, in query_http_devices
    devices = await http_get_devices(self.api_key)
  File "/usr/local/lib/python3.10/site-packages/govee_led_wez/http.py", line 52, in http_get_devices
    return [
  File "/usr/local/lib/python3.10/site-packages/govee_led_wez/http.py", line 60, in <listcomp>
    properties=d["properties"],
KeyError: 'properties'
```

Is that enough to diagnose? Is there more I can share? Is there something I'm missing?

**Comment by wez** [comment]
I'm assuming that one of those devices is missing `properties` and we need to react accordingly, so it would be great to see what we're dealing with!

Could you try manually curl'ing the HTTP endpoint to retrieve the device list, and sharing it here?

```bash
curl -H Govee-API-Key:$GOVEE_KEY "https://developer-api.govee.com/v1/devices"
```

**Comment by phil-the-dev** [comment]
Sure thing!

```
{
  "data": {
    "devices": [
      {
        "device": "DEVICE:ID:1",
        "model": "H6199",
        "deviceName": "DeviceName1",
        "controllable": true,
        "retrievable": true,
        "supportCmds": ["turn", "brightness", "color", "colorTem"],
        "properties": { "colorTem": { "range": { "min": 2000, "max": 9000 } } }
      },
      {
        "device": "DEVICE:ID:2",
        "model": "H6008",
        "deviceName": "DeviceName2",
        "controllable": true,
        "retrievable": true,
        "supportCmds": ["turn", "brightness", "color", "colorTem"],
        "properties": { "colorTem": { "range": { "min": 2700, "max": 6500 } } }
      },
      {
        "device": "DEVICE:ID:3",
        "model": "H6008",
        "deviceName": "DeviceName3",
        "controllable": true,
        "retrievable": true,
        "supportCmds": ["turn", "brightness", "color", "colorTem"],
        "properties": { "colorTem": { "range": { "min": 2700, "max": 6500 } } }
      },
      {
        "device": "DEVICE:ID:4",
        "model": "H6008",
        "deviceName": "DeviceName4",
        "controllable": true,
        "retrievable": true,
        "supportCmds": ["turn", "brightness", "color", "colorTem"],
        "properties": { "colorTem": { "range": { "min": 2700, "max": 6500 } } }
      },
      {
        "device": "DEVICE:ID:5",
        "model": "H6008",
        "deviceName": "DeviceName5",
        "controllable": true,
        "retrievable": true,
        "supportCmds": ["turn", "brightness", "color", "colorTem"],
        "properties": { "colorTem": { "range": { "min": 2700, "max": 6500 } } }
      },
      {
        "device": "DEVICE:ID:6",
        "model": "H5080",
        "deviceName": "DeviceName6",
        "controllable": true,
        "retrievable": true,
        "supportCmds": ["turn"]
      },
      {
        "device": "DEVICE:ID:7",
        "model": "H5080",
        "deviceName": "DeviceName7",
        "controllable": true,
        "retrievable": true,
        "supportCmds": ["turn"]
      }
    ]
  },
  "message": "Success",
  "code": 200
}

```

**Comment by phil-the-dev** [comment]
I believe it's the wifi plugs that were the new additions, that don't have `properties` is what borked it all

**Comment by wez** [comment]
I see; those `H5080` smart plugs only support on/off and not the other light controls.
I can make a quick fix to ignore those, but we could probably also support switches separately in the future.

**Comment by wez** [comment]
I pushed the minimum effort first pass at this, which is to just pretend that properties are empty when missing. You'll probably see your plugs show up as lights. Hopefully that will unblock the use of the actual lights.
This should be available to update already in HACS.

I don't know what will happen if you try to control the plugs as lights; it's possible that it might allow turning them on/off, but it may error out.  Please let me know!

**Comment by phil-the-dev** [comment]
I've updated my integration, and it works! And the plugs work too!

**Comment by wez** [comment]
Great! I've opened a separate issue to track making them show up as proper switches
